ztools/x509/extensions_test.go

```diff
@@ -48,7 +51,7 @@ func (s *ExtensionsSuite) SetUpTest(c *C) {
 
 func (s *ExtensionsSuite) TestEncodeDecodeSAN(c *C) {
 	for _, cert := range s.parsedCerts {
-		if (cert.Issuer.CommonName != "SAN Test"){
+		if cert.Issuer.CommonName != "SAN Test" {
```

Comment:
I'm not sure either, I only added the name constraints test. If we were to do the parsing inside of the test, we need to restructure the testing code. But again, I didn't come up them.